server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoShapeIntegrationIT.java

```diff
@@ -136,6 +136,34 @@ public void testIgnoreMalformed() throws Exception {
         assertThat(searchResponse.getHits().getTotalHits().value, equalTo(1L));
     }
 
+    /**
+     * Test that geo_shape field can parse input as an array of values.
+     */
+    public void testArrayOfGeoShapeParsing() throws Exception {
```

Comment:
This test could also instead live in `GeoShapeFieldMapperTests`, which is closer to a unit test suite. It seems common to test document parsing logic in `*FieldMapperTests`.

Reply:
will do!